### DISCUSSION TITLE
Community meetings

### DISCUSSION THREAD

**Discussion by ionut-arm** [parallaxsecond/rust-tss-esapi#418] [Ideas]
Hi all,

I've been thinking about the fact that we don't have a community meeting specific to this project (thank you @wiktor-k for planting the idea!). Perhaps towards the start of autumn we can get the ball rolling on that. Not sure yet what cadence, or the exact timing, but I just wanted to get a feel for whether people feel this might be a good use of our time. We could discuss anything from the status of open PRs to release schedules or features that would be useful.

Thoughts?

cc @Superhepper @wiktor-k @baloo @puiterwijk @hug-dev @THS-on @gowthamsk-arm

**Comment by THS-on** [comment]
I think this is good idea. For the start probably a monthly cadence is fine.

**Comment by Superhepper** [comment]
Yeah, I cannot imagine we would have so much to discuss that we would need more then that.

**Comment by ionut-arm** [comment]
A bit late, but would those interested please fill in [this](https://doodle.com/meeting/participate/id/enz34KYe) doodle poll? Trying to plan a first occurrence of this meeting :)

**Reply by ionut-arm** [reply]
@mkulke - would you (or anyone working with you) be interested in this meeting, from a CoCo/Azure CVM perspective?

**Reply by mkulke** [reply]
thanks for the headsup. i'd be interested and voted on the doodle
